tests/pf/assets/magnetics/Mesh_10m.msh

```diff
@@ -0,0 +1,5 @@
+47 47 20
```

Comment:
we should move all of the assets to a database @rowanc1 : is setting up a bucket on gae a longer time-scale? if so, any suggestions for a short term soln?